cmp/relativetimestamp/RelativeTimestamp.js

```diff
@@ -31,8 +31,23 @@ const FORMAT_STRINGS = {
     years: '%d years'
 };
 
+const SHORT_FORMAT_STRINGS = {
+    seconds: '<1 min',
+    minute: '1 min',
+    minutes: '%d mins',
+    hour: '~1 hour',
+    hours: '~%d hours',
+    day: 'a day',
+    days: '%d days',
+    month: 'a month',
+    months: '%d months',
+    year: 'a year',
+    years: '%d years'
+};
+
 const defaultOptions = {
     allowFuture: false,
+    short: false, // Defaulted based on device in getRelativeTimestamp method
```

Comment:
So this is because XH is not ready here outside of a component/model. We seem to be doing things here in an old way. And by 'old' I mean a year, software is crazy :).

I could see use putting together a Component Model relationship here, but not sure we want to invest the time in work on a dinky component thats already sort of doing what we want.